fideslib/oauth/oauth_util.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import annotations
+
+from datetime import datetime
+
+from jose import jwe
+
+from fideslib.core.config import config
```

Comment:
This feels like more unnecessary coupling; these utility/helper functions shouldn't be dependent on configuration.

Reply:
How else would you do it? There are values that are configurable so we need to get them in there somehow.

Reply:
I'd rather the functions accept arguments, and callers pass in values from a more localized `config` object, or similar.